crates/telio-starcast/src/multicaster.rs

```diff
@@ -0,0 +1,281 @@
+use async_trait::async_trait;
```

Comment:
NIT: in general i prefer to follow rust-lang code style for imports: https://doc.rust-lang.org/1.0.0/style/style/imports.html

Main thing beeing spliting std, 3rd-party creates, our crates, and mods into seperte groups